### PR TITLE
feat(content): Frozen climate news

### PR DESCRIPTION
**Pull request by dorbarker**
## Summary

Adds human news items that make use of the the `frozen` attribute added in #8880 
- arctic explorer
- mechanic [frozen]
- tourist [frozen]

### CODE REVIEW

File: data/human/news.txt [hunk 1]
```diff
@@ -1963,6 +1995,33 @@ news "republic navy advisory system [wartime]"
...
+		word
+			`"`
+		word
+			"There's only so much you can get from orbital mapping. Sometimes, you've got to gear up and go see it yourself."
+			"Cold? You don't know cold. You don't know true cold until you've been bivouacked in a scree field for a week, wondering when the storm will end."
+			"Happiness is finding a can of peaches you buried in the snow a month ago."
+			"It's my responsibility to check on the remote weather stations. If one of those go down, the whole colony could be at risk."
+			"There's no bad weather, only bad clothing."
+			"It lacks the romance of furs and seal skin, but I'm glad for modern textiles."
+			"It's hard to have a bond with a snowcat quite like you do with a husky."
```
Comment (lumbar527) [inline data/human/news.txt:2019]: Might not matter, but are there snowcats on other planets?

**Review by lumbar527 — APPROVED** [review]
Other than the one slight thing, looks good!

File: data/human/news.txt [hunk 2]
```diff
@@ -1463,6 +1479,25 @@ news "tourist"
...
+			"Arctic tourist"
+			"Tourist"
+	message
+		word
+			`"`
+		word
+			"Have you ever seen so much white?"
+			"This place has such a stark beauty."
+			"I know this isn't the usual sort of tourist destination, but I like getting off the beaten path."
+			"Enjoying a hot tub while in the bitter cold really is the sort of pleasure that just isn't available in the Paradise Worlds."
```
Comment (Zitchas) [inline data/human/news.txt:1498]: Worth noting that there *is* a paradise peripheral world famed for its winter sports and resorts. Chances are they probably have hot tubs there...

File: data/human/news.txt (hunk 1 above)
Comment (dorbarker) [reply data/human/news.txt:2022]: "[Snowcat](https://en.wikipedia.org/wiki/Snowcat)" is a deliberate play on words vis-a-vis a dog (husky) as arctic transport.

If the joke is too niche, I'm happy removing it, though.

**Review by Zitchas — COMMENTED** [review]
One slight detail that is off, but otherwise these look really good. Nice to get some additional variety into the news system.

File: data/human/news.txt (hunk 1 above)
Comment (lumbar527) [reply data/human/news.txt:2022]: I did not know what that is. I would wait for other people's input, though.

**Review by EjoThims — APPROVED** [review]
Good variety of additions :)

File: data/human/news.txt (hunk 1 above)
Comment (TheMarksman-ES) [reply data/human/news.txt:2022]: No no, leave it.

**Review by TheMarksman-ES — APPROVED** [review]
Nothing that seems off to me. Looking good.

File: data/human/news.txt [hunk 3]
```diff
@@ -890,6 +890,22 @@ news "mechanic"
 		word
 			`"`
 
+news "mechanic [frozen]"
```
Comment (Anarchist2) [inline data/human/news.txt:893]: Style.
```suggestion
news "mechanic on frozen world"
```

File: data/human/news.txt [hunk 4]
```diff
@@ -1463,6 +1479,25 @@ news "tourist"
 		word
 			`"`
 
+news "tourist [frozen]"
```
Comment (Anarchist2) [inline data/human/news.txt:1482]:
```suggestion
news "tourist on frozen world"
```

File: data/human/news.txt [hunk 5]
```diff
@@ -1963,6 +1998,33 @@ news "republic navy advisory system [wartime]"
 		word
 			`"`
 
+news "explorer [frozen]"
```
Comment (Anarchist2) [inline data/human/news.txt:2001]:
```suggestion
news "explorer on frozen world"
```

File: data/human/news.txt [hunk 6]
```diff
@@ -1963,6 +1998,33 @@ news "republic navy advisory system [wartime]"
...
+	location
+		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
+		attributes "frozen"
+	name
+		word
+			"Arctic"
+			"Glacial"
+		word
```
Comment (Anarchist2) [inline data/human/news.txt:2009]:
```suggestion
		word
			" "
		word
```

**Review by Zitchas — APPROVED** [review]
Looks good in terms of lore, flavor, and character. Thanks for working on this!

**Review by OcelotWalrus — APPROVED** [review]
Looks like it's a good add-on!

File: data/human/news.txt [hunk 7]
```diff
@@ -1963,6 +1998,35 @@ news "republic navy advisory system [wartime]"
...
 			`"`
 
+news "explorer on frozen world"
+	location
+		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
+		attributes "frozen"
+	name
+		word
+			"Arctic"
+			"Glacial"
```
Comment (bene-dictator) [inline data/human/news.txt:2008]:
```suggestion
			"Glacial"
			"Polar"
```

**Review by bene-dictator — APPROVED** [review]
Nice! One optional suggestion.

**Review by Quantumshark — APPROVED** [review]

**Pull request merged**